changelog/README.md

```diff
@@ -47,11 +45,11 @@ Every push to the main or release branch Assemble changelog GitHub action will b
 Some other changes
 ```
 
-* write the changelog to the `changelog/unreleased/CHANGELOG.md` file
+To view the changelog for the android auto project, run the command `python3 scripts/changelog/assemble_changelog.py --auto`.
 
 Every release the release train app will:
 
-* get changelog from `changelog/unreleased/CHANGELOG.md` file
+* assemble the changelog by the script `python3 scripts/changelog/assemble_changelog.py`
```

Comment:
If you add `#!/usr/bin/env python3` to the top of the file, you can run script typing less 🙂 

```suggestion
* assemble the changelog by the script `./scripts/changelog/assemble_changelog.py`
```